spec/migrations/20180130165200_drop_webmks_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_migration
+
+describe DropWebmks do
+  let(:server_stub)   { migration_stub(:MiqServer) }
```

Comment:
This is failing rspec since `class MiqServer < ActiveRecord::Base; end ` isn't defined in the migration namespace